### DISCUSSION TITLE
Pass  {...register('name')} to custom component but not work.

### DISCUSSION THREAD

**Discussion by IRediTOTO** [react-hook-form/react-hook-form#6462] [Q&A] [answered]
Here is the codesandbox :https://codesandbox.io/embed/react-hook-form-defaultvalues-v7-forked-5kueu?fontsize=14&hidenavigation=1&theme=dark

I just trying to pass register to children

**Comment by bluebill1049** [comment, marked as answer]
make sure you forward the ref:

https://reactjs.org/docs/forwarding-refs.html

**Reply by IRediTOTO** [reply]
thank you so much.
